linux/user-and-file-management/linux-intro/linux-common-terminology.md

```diff
@@ -0,0 +1,60 @@
+---
+author: kapnobatai136
+
+aspects:
+  - introduction
+
+type: normal
+
+category: must-know
+
+tags:
+  - introduction
+  - linux
+
+---
+
+# Common Terminology
+
+---
+## Content
+
+
+Let's go over a few terms that you'll see a lot in this course.
+
+A **command** is just a piece of text that you type into the **terminal**. Think of this as an instruction that you give to your computer.
+
+Sometimes you might type some commands which are restricted. That's where the **root user** comes in. It's the unique admin user, with superpowers. Unsurprisingly, it's also called the **superuser**.
+
+The **package manager** removes the tedious process of installing or uninstalling apps. Think of a **package** as being an app.
+
+Remember the terminal? Well, this is also referred to as the **shell**. 
+
+Linux comes with an improved version of the shell, which is called **bash**[1] (**Bourne-Again SHell**). From now on, we'll be using these terms interchangeably.
+
+> **Note:** as with all the skills in the app, if you want to access more advanced or specific content at any time, **feel free to use the *Skill Map* tab** to jump to the workouts most relevant for you.
+
+---
+## Practice
+
+Commands are typed in the ???.
+
+* terminal
+* package manager
+* superuser manager
+
+---
+## Revision
+
+The **root user** is also called the ???.
+
+* superuser
+* extrauser
+* super
+* best-user
+
+---
+## Footnotes
+
+[1:Bash]
+The name is an acronym for the ‘Bourne-Again SHell’, a pun on Stephen Bourne, the author of the direct ancestor of the current Unix shell `sh`, which appeared in the Seventh Edition Bell Labs Research version of Unix.
```

Comment:
The quotes are wrong:

```suggestion
The name is an acronym for the 'Bourne-Again SHell', a pun on Stephen Bourne, the author of the direct ancestor of the current Unix shell `sh`, which appeared in the Seventh Edition Bell Labs Research version of Unix.
```

Reply:
Also, we suddenly introduce Unix here without ever mentioning it before. This would seem like a typo to most users unfamiliar with it (and since we're in the Linux topic)

```suggestion
The name is an acronym for the ‘Bourne-Again SHell’, a pun on Stephen Bourne, the author of the direct ancestor of the another shell called `sh`.
```